hw/ip/edn/data/edn.hjson

```diff
@@ -254,7 +256,41 @@
       desc: "Recoverable alert status register",
       swaccess: "rw0c",
       hwaccess: "hwo",
+          tags: [// Internal HW can modify status register
+                 "excl:CsrAllTests:CsrExclCheck"]
       fields: [
+        { bits: "0",
+          name: "EDN_ENABLE_FIELD_ALERT",
+          desc: '''
+                This bit is set when the EDN_ENABLE field is set to an illegal value,
+                something other than 0x5 or 0xA.
+                Writing a zero to this register resets the status bits.
```

Comment:
Sorry for this nit post-merge @mwbranstad / @tjaychen - is it more correct for this to say that writing a zero to this *field* (or bit, your choice) resets the bit, rather than "bits" as it says now?  The way this is written it sort of suggests to me that writing a zero to any of the bits 0-3 resets all of them, which I don't think is what the logic does, and would be a surprising result.

I'm not sure how consistent we are with the phrasing between register and field throughout the docs, but this one just stuck out to me.

Reply:
ah yeah chris i think you're right.

Reply:
I saw this before as well, and decided to leave it, but probably should be written to be a on bit basis only. Will create a PR to fix both EDN and CSRNG.